### PR TITLE
Upgraded tensorflow version

### PR DESCRIPTION
**Pull request by TheCaffeineDev**
Upgraded TensorFlow nightly version to Tensorflow 2.1.0

### CODE REVIEW

File: examples/colab/object_detection.ipynb [hunk 1]
```diff
@@ -80,20 +99,18 @@
...
-      "outputs": [],
       "source": [
         "#@title Imports and function definitions\n",
         "\n",
-        "# Currently %tensorflow_version 2.x installs beta1, which doesn't work here.\n",
-        "# %tensorflow_version can likely be used after 2.0rc0  \n",
-        "!pip install tf-nightly-gpu-2.0-preview\n",
+        "# Now stable version tensorflow 2.1.0 is there\n",
+        "\n",
+        "!pip install tensorflow==2.1.0\n",
```
Comment (vbardiovskyg) [inline examples/colab/object_detection.ipynb]: Does this still enable running on a GPU kernel? You can test by switching the kernels and measuring the time it takes. My guess is that tensorflow-gpu has to be installed.

Comment (TheCaffeineDev) [reply examples/colab/object_detection.ipynb]: There is no separate installation of TensorFlow GPU in 2.1.0 version. Now _pip install tensorflow_ automatically installing both CPU and GPU version. And I just checked also, the kernel is running on GPU.

Comment (TheCaffeineDev) [reply examples/colab/object_detection.ipynb]: Reference link - https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0

Comment (vbardiovskyg) [reply examples/colab/object_detection.ipynb]: Sounds good, thanks for checking!

File: examples/colab/object_detection.ipynb [hunk 2]
```diff
@@ -80,20 +99,18 @@
...
+        "colab": {}
       },
-      "outputs": [],
       "source": [
         "#@title Imports and function definitions\n",
         "\n",
-        "# Currently %tensorflow_version 2.x installs beta1, which doesn't work here.\n",
-        "# %tensorflow_version can likely be used after 2.0rc0  \n",
-        "!pip install tf-nightly-gpu-2.0-preview\n",
+        "# Now stable version tensorflow 2.1.0 is there\n",
```
Comment (vbardiovskyg) [inline examples/colab/object_detection.ipynb]: Could you please change this line to:

# Runs with stable version tensorflow 2.1.0.

Comment (TheCaffeineDev) [reply examples/colab/object_detection.ipynb]: I have now changed the line. Thanks for reviewing.

**Comment by vbardiovskyg** [comment]
Thanks for the contribution! Your PR has been merged at 27981849c64aa06398af3768949b6edd8e1f8535.

**Pull request closed without merging**